File: app/src/main/java/cc/zgeek/page/Lib/Annotation/InjectView.java
package cc.zgeek.page.Lib.Annotation;

import android.support.annotation.IdRes;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by flyop.
 * Change History:
 * 2017/1/14 : Create
 */

@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.FIELD)
public @interface InjectView {
    @IdRes int value();
}

File: pagelib/src/main/kotlin/cc/zgeek/pagelib/Annotation/InjectView.kt
package cc.zgeek.pagelib.Annotation

import android.support.annotation.IdRes
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

/**
 * Created by ZGeek.
 * Change History:
 * 2017/1/14 : Create
 */

@Retention(RetentionPolicy.RUNTIME)
@Target(AnnotationTarget.FIELD)
annotation class InjectView(@IdRes val value: Int)
